platform/ui/src/components/DateRange/DateRange.jsx

```diff
@@ -60,35 +76,30 @@ const DateRange = props => {
     );
   };
   const renderMonthElement = ({ month, onMonthSelect, onYearSelect }) => {
-    const renderYearsOptions = () => {
-      const yearsRange = 20;
-      const options = [];
-
-      for (let i = 0; i < yearsRange; i++) {
-        const year = moment().year() - i;
-        options.push(
-          <option key={year} value={year}>
-            {year}
-          </option>
-        );
-      }
-
-      return options;
-    };
-
     renderMonthElement.propTypes = {
       month: PropTypes.object,
       onMonthSelect: PropTypes.func,
       onYearSelect: PropTypes.func,
     };
 
+    const handleMonthChange = event => {
+      onMonthSelect(month, event.target.value);
+    };
+
+    const handleYearChange = event => {
+      onYearSelect(month, event.target.value);
+    };
+
+    const handleOnBlur = () => {};
```

Comment:
why it's calling an empty function?

Reply:
Our eslint has a rule that requires us to add an onBlur on <select> components. 
As we don't do anything onBlur I just create the empty function. 

We could change the the eslint rules.